File: Domain/Models/CommandeLigne.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Domain.Models 
{ 
    public class CommandeLigne
    {
        [Key]
        public int IdCmdLigne { get; set; }
        public decimal PrixProd { get; set; }
        public DateTime DateLigne { get; set; }
        public decimal QteLigne { get; set; }
        public int IdCmdEnt { get; set; }
        public CommandeEntete CommandeEntete { get; set; }
        public Produit Produit { get; set; }
        public int IdProd { get; set; }

    }
}

File: angulacommande/src/app/Model/commande-ligne.ts
import { CommandeEntete } from "./commande-entete";
import { Produit } from "./produit";

export class CommandeLigne {
    idCmdLigne: number=0;
    prixProd: number=0;
    dateLigne: string="";
    qteLigne: number=0;
    idCmdEnt: number=0;
    commandeEntete: CommandeEntete = new CommandeEntete;
    produit: Produit = new Produit;
    idProd: number=0;

}
